phoenix-core/src/main/java/org/apache/phoenix/iterate/ExplainTable.java

```diff
@@ -215,7 +215,7 @@ private void appendPKColumnValue(StringBuilder buf, byte[] range, Boolean isNull
     private Long getViewIndexValue(PDataType type, byte[] range) {
         boolean useLongViewIndex = MetaDataUtil.getViewIndexIdDataType().equals(type);
         Object s = type.toObject(range);
-        return (useLongViewIndex ? (Long) s : (Short) s) + (useLongViewIndex ? Long.MAX_VALUE : Short.MAX_VALUE) + 2;
+        return (useLongViewIndex ? (Long) s : (Short) s) + Short.MAX_VALUE + 2;
```

Comment:
I guess this isn't relevant to your change, but is the `+2` for the 'L#' characters? If yes, can we change this to `LOCAL_INDEX_COLUMN_FAMILY_PREFIX.length` for better readability?

Reply:
I actually don't know. :( I just made sure it behaves like before.